src/html/_style_base.html.template

```diff
@@ -1,4 +1,4 @@
-<meta name='viewport' content='width=device-width, user-scalable=no'>
+<meta name='viewport' content='width=device-width'>
```

Comment:
I see the header doesn't deal with this very well, but other than that it seems to work ok...

Reply:
Seems to work okay for me; I've been using this in my HA install for several months.